gnomad_qc/v4/create_release/create_de_novo_release.py

```diff
@@ -42,343 +31,248 @@
-        vep_globals=ht.vep_globals.annotate(
-            gencode_version=GENCODE_VERSION,
-            mane_select_version=MANE_SELECT_VERSION,
-        ),
-        tool_versions=ht.tool_versions.annotate(
-            dbsnp_version=DBSNP_VERSION,
-            sift_version=SIFT_VERSION,
-            polyphen_version=POLYPHEN_VERSION,
-        ),
-        date=datetime.now().isoformat(),
-        version=CURRENT_RELEASE,
-        frequency_README=get_freq_array_readme(data_type="exomes"),
+    ht = ht.filter(
+        ht.de_novo_AC.AC_high_conf + ht.de_novo_AC.AC_medium_conf_P_0_9 <= 15
+    )
+    logger.info(
+        f"Getting {ht.count()} high-quality coding de novos with AC <= 15 in the "
+        f"callset..."
     )
 
-    # Organize the fields in the release HT to match the order of FINALIZED_SCHEMA when
-    # the fields are present in the HT.
-    final_fields = get_final_ht_fields(ht, schema=FINALIZED_SCHEMA)
-    return ht.select(*final_fields["rows"]).select_globals(*final_fields["globals"])
+    return ht_all_hq, ht
 
 
 def restructure_for_tsv(ht: hl.Table) -> hl.Table:
     """
     Restructure the de novo release HT for TSV export.
 
+    .. note::
+       This function will only be used for the TSV of high confidence coding de novos.
+
     :param ht: De novo release HT.
     :return: Restructured HT.
     """
-    # TODO: Change when we decide on the final de novo data to release.
-    ht = ht.filter((ht.filters.length() == 0) & (ht.de_novo_stats.n_de_novo > 0))
-    ht = process_consequences(ht, penalize_flags=False, has_polyphen=False)
-    return (
-        ht.select(
-            "de_novo_stats",
-            exomes=ht.exomes_freq[0],
-            genomes=ht.genomes_freq[0],
-            joint=ht.joint_freq[0],
-            worst_csq_for_variant_canonical=ht.vep.worst_csq_for_variant_canonical.select(
-                "most_severe_consequence",
-                "transcript_id",
-                "gene_id",
-                "gene_symbol",
-                "mane_select",
-                "lof",
-                "hgnc_id",
-            ),
-            **ht.in_silico_predictors,
-        )
-        .flatten()
-        .rename(
-            {
-                "de_novo_stats.n_de_novo": "n_de_novo",
-                "de_novo_stats.parents.AC": "trio_parents.AC",
-                "de_novo_stats.parents.AN": "trio_parents.AN",
-                "de_novo_stats.children.AC": "trio_children.AC",
-                "de_novo_stats.children.AN": "trio_children.AN",
-            }
-        )
+    return ht.flatten().rename(
+        {
+            "de_novo_AC.AC_all": "de_novo_AC_all",
+            "de_novo_AC.AC_all_raw": "de_novo_AC_all_raw",
+            "de_novo_AC.AC_high_conf": "de_novo_AC_high_conf",
+            "de_novo_AC.AC_medium_conf": "de_novo_AC_medium_conf",
+            "de_novo_AC.AC_medium_conf_P_0_9": "de_novo_AC_medium_conf_P_0_9",
+            "de_novo_AC.AC_low_conf": "de_novo_AC_low_conf",
```

Comment:
this TSV should contain at most the high confidence and medium (p > whatever threshold is decided) de novos, so you should drop the medium and low confidence fields rather than renaming them